borsh/tests/test_simple_structs.rs

```diff
@@ -100,6 +100,7 @@ struct F2<'b> {
 }
 
```

Comment:
This is a minimal test of the subject of this pr. `false` value for the flag
doesn't behave as borsh before 0.11.0, test `test_discriminant_minimal_false` is failing. 
```rust
// minimal
#[derive(BorshSerialize)]
#[borsh(use_discriminant = true)]
enum MyDiscriminantEnum {
    A = 20,
}

#[derive(BorshSerialize)]
#[borsh(use_discriminant = false)]
enum MyDiscriminantEnumFalse {
    A = 20,
}

#[derive(BorshSerialize)]
enum MyEnumNoDiscriminant {
    A,
}
#[test]
fn test_discriminant_minimal_true() {
    assert_eq!(MyDiscriminantEnum::A as u8, 20);
    assert_eq!(MyDiscriminantEnum::A.try_to_vec().unwrap(), vec![20]);
    
}

#[test]
fn test_discriminant_minimal_false() {
    assert_eq!(MyDiscriminantEnumFalse::A as u8, 20);
    assert_eq!(
        MyEnumNoDiscriminant::A.try_to_vec().unwrap(),
        MyDiscriminantEnumFalse::A.try_to_vec().unwrap(),
    );    
    assert_eq!(MyDiscriminantEnumFalse::A.try_to_vec().unwrap(), vec![0]);

}

```